dspace-server-webapp/src/test/java/org/dspace/app/rest/TiltedRelationshipRestRepositoryIT.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest;
+
+import org.dspace.content.RelationshipType;
+import org.junit.Before;
+
+/**
+ * This class carries out the same test cases as {@link RelationshipRestRepositoryIT}.
+ * The only difference being that a RelationshipType is set on
+ * {@link RelationshipRestRepositoryIT#isAuthorOfPublicationRelationshipType}.
+ */
+public class TiltedRelationshipRestRepositoryIT extends RelationshipRestRepositoryIT {
```

Comment:
This seems to prove that the results from the parent test are not affected. This is most likely because the parent test doesn't verify the metadata on the Person item in the isAuthorOfPublication relationship.

This seems a bit confusing, and it may just as well pass if the setup method doesn't work correctly. So it would be good to add an additional test to the parent which verifies the metadata on the Person item in the isAuthorOfPublication relationship. Something similar to TiltedRelationshipMetadataServiceIT, but verifying REST also complies to these changes

Reply:
This looks good to me now